contracts/SportMarkets/SportPositions/SportPositionalMarketManager.sol

```diff
@@ -557,6 +661,32 @@ contract SportPositionalMarketManager is Initializable, ProxyOwned, ProxyPausabl
         emit DatesUpdatedForMarket(_market, _newStartTime, _expiry);
     }
 
+    function _hasAnyMintsForChildren(address _market) internal view returns (bool) {
+        return _hasAnyMintsForHT(_market) || _hasAnyMintsForPP(_market);
+    }
+
+    function _hasAnyMintsForHT(address _market) internal view returns (bool) {
```

Comment:
If the contract size needs to be reduced, these two methods can be written as one as along as IGamesOddsObtainer and IGamesPlayerProps have a common interface declaring the methods used here

Reply:
updated